packages/actor-query-operation-group/package.json

```diff
@@ -36,7 +36,7 @@
     "@comunica/bus-query-operation": "^2.0.1",
     "@comunica/core": "^2.0.1",
     "@comunica/types": "^2.0.1",
-    "@types/json-stable-stringify": "^1.0.32",
+    "@rdfjs/types": "^1.1.0",
```

Comment:
Rather than detecting if it is a type-only package, why don't we detect if there is a resolution defined in the global package.json? https://github.com/comunica/comunica/blob/448bfbadd805dcad95841789cd12b0d33a672585/package.json#L91-L93

Reply:
Sure, that might be fine for now.
In the long-term, we probably want this for _all_ type packages (not just the ones we're overriding via resolutions), but that's not very urgent atm.